src/Symfony/Component/DependencyInjection/Tests/Fixtures/yaml/services9.yml

```diff
@@ -76,6 +76,9 @@ services:
         class: stdClass
         decorates: decorated
         decoration_inner_name: decorated.pif-pouf
+    deprecated_service:
+        class: stdClass
+        deprecated: The "%service_id%" service is deprecated. You should stop using it, as it will soon be removed.
```

Comment:
Can you add test with `deprecated: true` (without overwrite default deprecation message)?